src/bluesearch/entrypoint/database/convert_pdf.py

```diff
@@ -99,9 +99,12 @@ def run(
     able to combine the functions in this way:
 
     >>> import argparse
-    >>> parser = init_parser(argparse.ArgumentParser())
-    >>> args = parser.parse_args()
-    >>> run(**vars(args))
+    >>> from bluesearch.entrypoint.database import convert_pdf
+    >>> parser = convert_pdf.init_parser(argparse.ArgumentParser())
+    >>> # replace with true values and uncomment
+    >>> argv = ["host", "port", "pdf_path", "xml_path"]
+    >>> # args = parser.parse_args(argv)
+    >>> # convert_pdf.run(**vars(args))
```

Comment:
Are those changes needed in this PR ?

Reply:
Yes, the doctests were failing... I had to comment out some lines because I couldn't come up with a runnable example (one of the parameters to the argument parser must be a file that exists on the disk...) I'd be super happy to hear suggestions how to improve this one!